crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -34,6 +34,7 @@ def upgrade
       case @version
       when "1.9"
         knife_files_1_9
+        reset_target_platform_for_nodes
```

Comment:
I can't see how this isn't a behaviour change.  You're calling a method which wasn't called before, and nothing else in this commit relates to that.